js/interface.js

```diff
@@ -555,6 +571,217 @@ function uploadFiles(files) {
+function addAppsToSelect(apps) {
+  var $appSelectList = $('#organization-optgroup-' + apps.organizationId);
+  $appSelectList.append(templates.appsSelect(apps));
+}
+
+// Create list of folders in modal
+function addFoldersToMoveModal(res) {
+  if (Array.isArray(res)) {
+    if (res.length < 1) {
+      $('.move-modal-empty-state').addClass('active');
+    } else {
+      res = _.sortBy(res, [function(o) {
+        return o.name;
+      }]);
+      res.forEach(function (folder) {
+        $('.move-modal-list').append(templates.moveModalFolders(folder));
+      });
+    }
+  } else {
+    $('.move-modal-list').append(templates.moveModalFolders(res));
+    $('.move-modal-empty-state').removeClass('active');
+  }
+}
+
+// Add organization select item template
+function addOrganizationsToSelect(org) {
+  var $organizationsSelectList = $('#move-model-organizations-select');
+  $organizationsSelectList.append(templates.organizationsSelect(org));
+  appsList.forEach(addAppsToSelect);
+}
+
+// Users selected folders tree (for breadcrumbs)
+function addPathToStack(id, name, orgId, appId, parentId, type) {
+  foldersForMoving['navStack'].push(
+    {
+      id: Number(id)?Number(id):null,
+      name: name,
+      organizationId: Number(orgId)?Number(orgId):null,
+      appId: Number(appId)?Number(appId):null,
+      parentId: Number(parentId)?Number(parentId):null,
+      type: type
+    }
+    );
+}
+
+//Check if any folders exists
+function checkExistsMovingFolders(id) {
+  if (foldersForMoving.hasOwnProperty(id)) {
+    return foldersForMoving[id];
+  } else {
+    return false;
+  }
+}
+
+//Function for creating breadcrumbs
+function createMoveModalBreadCrumbsPaths() {
+  if (foldersForMoving['navStack'].length >= 1) {
+    var breadcrumbsPath = '';
+    
+    for (var i = 0; i < foldersForMoving['navStack'].length; i++) {
+      breadcrumbsPath +=
+        '<span class="bread-link">' +
+        '<a href="#" ' +
+        'data-breadcrumb-item ' +
+        'data-breadcrumb="' + foldersForMoving['navStack'][i].id + '" ' +
+        'data-breadcrumb-index="' + i + '" ' +
+        'data-breadcrumb-type="' + foldersForMoving['navStack'][i].type + '"' +
+        '>' + foldersForMoving['navStack'][i].name + '</a>' +
+        '</span>';
+    }
+    
+    $('#move-modal .current-folder-title').html(breadcrumbsPath);
+  }
+}
+
+//Create array tree form search result
+function createTreeFoldersArr(arr) {
+  var currentArr = arr;
+  
+  function transformToTree(arr) {
+    var nodes = {};
+    return arr.filter(function(obj) {
+      var id = obj["id"],
+        parentId = obj["parentId"];
+      
+      nodes[id] = _.defaults(obj, nodes[id], { children: [] });
+      parentId && (nodes[parentId] = (nodes[parentId] || { children: [] }))["children"].push(obj);
+      
+      return !parentId;
+    });
+  }
+  return transformToTree(currentArr);
+}
+
+//Create and open move modal
+function openMovePopup() {
+  foldersForMoving['navStack'] = [];
```

Comment:
please add an empty line before the first block of each `var` declaration and also to separate different  logics, e.g.:

```
$(foo).bar

var a;
var b;

$(something).else()

var c
var d
var e

addPathToStack()

this.searchFOlders()
```

Reply:
fixed